test/e2e/storage/csi_volumes.go

```diff
@@ -95,11 +95,9 @@ var _ = Describe("PMEM Volumes", func() {
 					},
 
 					Config: testsuites.TestConfig{
-						Framework: f,
-						Prefix:    "pmem",
-						// Ensure that all pods land on the same node. Works around
-						// https://github.com/intel/pmem-csi/issues/132.
-						ClientNodeName: "host-1",
+						Framework:       f,
+						Prefix:          "pmem",
+						TopologyEnabled: true,
```

Comment:
Which commit is getting fixed here? Better squash it into that one.

Overall I'm getting the impression that the following commits are all related. Squash them?
- Implemented cache persistency model
- fix e2e tests
- ReadME.md: updated documentation with different persistency models.

What I don't find is the commit that removes (or updates?) persistent volume support. Are you going to do that later?

Reply:
Squashed the commits. Done

Reply:
> What I don't find is the commit that removes (or updates?) persistent volume support. Are you going to do that later?

Persistent Volume model is fixed as part of "Cache" change. And updated the diagram.